File: Java-Fundamentals-main/08 - Array 2/Selection Sort.java
public class Solution {  

    public static void selectionSort(int[] arr) {
    	//Your code goes here
        
        int n = arr.length;
        
        for(int i=0;i<n-1;i++)
        {
            int min = i;
            
            for(int j=i+1;j<n;j++)
            {
                if(arr[j]<arr[min])
                    min = j;
            }
            
            int tmp = arr[i];
            arr[i] = arr[min];
            arr[min] = tmp;
            
        }
        
    }   

}

File: Popular interview DSA questions in C++/Selection Sort.cpp
// https://www.codingninjas.com/codestudio/problems/selection-sort_981162


#include <bits/stdc++.h> 
void selectionSort(vector<int>& arr, int n)
{   
    // Write your code here.
    for(int i = 0; i < n-1; i++){
        int minIndex = i;
        
        for(int j = i+1; j<n;j++){
            if(arr[j]<arr[minIndex])
                minIndex = j;
        }
        swap(arr[minIndex],arr[i]);
    }
}